http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20240117_create_app_chart_version_table.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="leonardo" xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet logicalFilePath="leonardo" author="ladieu" id="create_app_chart_version_table">
+        <createTable tableName="APP_CHART_VERSION">
```

Comment:
@LizBaldo Calling out `APP_CHART_VERSION` name slightly different than what was in original design doc, but it sounded from our conversation that this might be a better name anyway.  LMK if you have other ideas.

Reply:
🤔 good point, this is less confusing indeed, and better reflects the fact that there is a row per chart, not per app :)